bot.js

```diff
@@ -102,10 +102,11 @@ bot.on('message', msg => {
   // Rank gold command
   else if (msg.content.startsWith(`${config.trigger}rank gold`)) {

```

Comment:
Also, as of right now, there are no parameters for the command. Instead of `startsWith()`, just check if the message content is `$rank`.
`msg.content === ...`